pyt/interprocedural_cfg.py

```diff
@@ -346,7 +349,15 @@ def visit_Call(self, node):
         self.function_return_stack.append(_id)
 
         local_definitions = self.module_definitions_stack[-1]
-        definition = local_definitions.get_definition(_id)
+
+        real_id = _id
+        for k in local_definitions.import_alias_mapping.keys():
+            if _id.startswith(k):
+                real_id = _id.replace(k, local_definitions.import_alias_mapping[k])
+                definition = local_definitions.get_definition(real_id)
```

Comment:
This makes sure, where `import bar as foo` if it sees foo it gets the definition of bar.